### PR TITLE
MULE-19751: Review dependencies of modules used as libs by tools

### PR DESCRIPTION
**Pull request by elrodro83**

### CODE REVIEW

**Comment by elrodro83** [comment]
--validate

**Comment by elrodro83** [comment]
--validate

**Comment by elrodro83** [comment]
--validate

File: core/src/main/java/org/mule/runtime/core/internal/connection/ConnectionUtils.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public static RetryPolicyTemplate getRetryPolicyTemplate(Optional<ReconnectionCo
 
   public static <C> void logPoolStatus(Logger logger, GenericObjectPool<C> pool, String poolId) {
     if (logger.isDebugEnabled()) {
-      String maxActive = pool.getMaxActive() < 0 ? "unlimited" : String.valueOf(pool.getMaxActive());
+      String maxActive = pool.getMaxTotal() < 0 ? "unlimited" : String.valueOf(pool.getMaxTotal());
```
Comment (fsgonz) [inline core/src/main/java/org/mule/runtime/core/internal/connection/ConnectionUtils.java:62]: is this ok? Why call it in one case maxTotal andin another maxActive?

**Review by fsgonz — DISMISSED** [review]

**Review by fsgonz — APPROVED** [review]

**Pull request merged**